api/v1/server/handlers/tenants/create_invite.go

```diff
@@ -29,18 +29,21 @@ func (t *TenantService) TenantInviteCreate(ctx echo.Context, request gen.TenantI
 	if apiErrors, err := t.config.Validator.ValidateAPI(request.Body); err != nil {
 		return nil, err
 	} else if apiErrors != nil {
+		t.config.Logger.Error().Msg("invalid request")
```

Comment:
Why are we logging at an `Error()` level for things that are returning a 400-level response to the user?